lib/AlipayAccount.php

```diff
@@ -64,8 +65,9 @@ public static function retrieve($_id, $_opts = null)
      */
     public static function update($_id, $_params = null, $_options = null)
     {
-        $msg = "Alipay accounts cannot be accessed without a customer ID. " .
-               "Call save() on \$customer->sources->retrieve('alipay_account_id') instead.";
+        $msg = "Alipay accounts cannot be updated without a customer ID. " .
+               "Update an Alipay account using `Customer::updateSource(" .
```

Comment:
Why doesn't it say `\Stripe\Customer` instead? Aren't most people using it that way?

Reply:
🤷‍♂  Depends on whether they have `using Stripe;` at the top of their code or not. I think it's clear enough from context that the class is namespaced under `\Stripe`.